spec.html

```diff
@@ -23338,6 +23337,12 @@ <h1>The Global Object</h1>
   <emu-clause id="sec-value-properties-of-the-global-object">
     <h1>Value Properties of the Global Object</h1>
 
+    <emu-clause id="sec-global-this">
```

Comment:
Sorry, now I'm thinking this clause shouldn't be "Value Properties" at all, it should be in "Other Properties".

The term "value property" isn't defined, but it seems to be used as a shorthand for "primitive value property", so `globalThis` doesn't belong.

Reply:
I don’t read that at all - it *happens* to be just primitives, but “value” to me doesn’t imply that. @allenwb do you have context here on the intention here between “value” and “other”?

Reply:
(The term "value" doesn't imply "primitive value" to me either, I'm just saying that *seems* to be the criterion of division.)

Reply:
(BTW, the heading "Value properties of the global object" goes all the way back to ES1.)

Reply:
My reading was that "Other Properties" was for namespaces and "Value Properties" was for everything else.

Maybe best to just merge the two sections.

Reply:
Hmm - "Other" contains only namespaces that are referenced elsewhere - maybe that's the distinction?

Whether they're merged or not, I think "Value Properties" is the proper place for it.

Reply:
Okay, plausible enough for me.